### PR TITLE
webrtc: use 'kind' instead of 'type'

### PR DESCRIPTION
**Pull request by fippo**
spec change in https://github.com/w3c/webrtc-pc/pull/1686

@adam-be @alvestrand PTAL

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21977)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/318529417)

File: webrtc/RTCPeerConnection-createOffer-offerToReceive.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
...
 
-  // Run some tests for both audio and video types
-  ['audio', 'video'].forEach((type) => {
-    const capsType = type[0].toUpperCase() + type.slice(1);
+  // Run some tests for both audio and video kinds
+  ['audio', 'video'].forEach((kind) => {
+    const capsType = kind[0].toUpperCase() + kind.slice(1);
 
     const offerToReceiveTrue = {};
     offerToReceiveTrue[`offerToReceive${capsType}`] = true;
```
Comment (adam-be) [inline webrtc/RTCPeerConnection-createOffer-offerToReceive.html]: You might want to update `capsType` as well

**Review by adam-be — COMMENTED** [review]
In any other context, 'type' feels more natural for me. :) But in WebRTC we usually call these 'kind' so it's good to be consistent.

**Review by adam-be — APPROVED** [review]
:+1:

**Review by alvestrand — APPROVED** [review]

**Pull request merged**